cmd/minikube/cmd/config/prompt.go

```diff
@@ -55,9 +56,14 @@ func AskForYesNoConfirmation(s string, posResponses, negResponses []string) bool
 }
 
 // AskForStaticValue asks for a single value to enter
-func AskForStaticValue(s string) string {
+func AskForStaticValue(s string, isOptional ...bool) string {
 	reader := bufio.NewReader(os.Stdin)
 
+	optional := false
+	if len(isOptional) > 0 {
```

Comment:
@andytom I just wand to add a default parameter to this method, and do not modify existed code, this is a kind of golang way.